src/config/roles.php

```diff
@@ -112,12 +112,12 @@
     'rolesGuiMiddleware'            => env('ROLES_GUI_MIDDLEWARE', 'role:admin'),
 
     // User Permissions or Role needed to create a new role
-    'rolesGuiCreateNewRolesMiddlwareType'   => env('ROLES_GUI_CREATE_ROLE_MIDDLWARE_TYPE', 'role'), //permissions or roles
-    'rolesGuiCreateNewRolesMiddlware'       => env('ROLES_GUI_CREATE_ROLE_MIDDLWARE_TYPE', 'admin'), // admin, XXX. ... or perms.XXX
+    'rolesGuiCreateNewRolesMiddlewareType'   => env('ROLES_GUI_CREATE_ROLE_MIDDLEWARE_TYPE', 'role'), //permissions or roles
+    'rolesGuiCreateNewRolesMiddleware'       => env('ROLES_GUI_CREATE_ROLE_MIDDLEWARE_TYPE', 'admin'), // admin, XXX. ... or perms.XXX
 
     // User Permissions or Role needed to create a new permission
-    'rolesGuiCreateNewPermissionMiddlwareType'  => env('ROLES_GUI_CREATE_PERMISSION_MIDDLWARE_TYPE', 'role'), //permissions or roles
-    'rolesGuiCreateNewPermissionsMiddlware'     => env('ROLES_GUI_CREATE_PERMISSION_MIDDLWARE_TYPE', 'admin'), // admin, XXX. ... or perms.XXX
+    'rolesGuiCreateNewPermissionMiddlewareType'  => env('ROLES_GUI_CREATE_PERMISSION_MIDDLEWARE_TYPE', 'role'), //permissions or roles
+    'rolesGuiCreateNewPermissionsMiddleware'     => env('ROLES_GUI_CREATE_PERMISSION_MIDDLEWARE_TYPE', 'admin'), // admin, XXX. ... or perms.XXX
```

Comment:
When looking closer at these lines; isn't only the first of the paired lines supposed to have the `_TYPE` suffix?